src/main/java/seedu/address/model/person/Remark.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+public class Remark {
```

Comment:
Consider to have one blank line before declare the class! (or because we will have JavaDocs comment here anyway)

Reply:
Thanks for the suggestion, will add it soon